fendermint/vm/topdown/src/finality/null.rs

```diff
@@ -73,15 +73,8 @@ impl FinalityWithNull {
         maybe_payload: Option<ParentViewPayload>,
     ) -> StmResult<(), Error> {
         if let Some((block_hash, validator_changes, top_down_msgs)) = maybe_payload {
-            emit!(
-                EventType::NewParentView,
-                is_null = false,
-                height,
-                block_hash = hex::encode(&block_hash)
-            );
             self.parent_block_filled(height, block_hash, validator_changes, top_down_msgs)
         } else {
-            emit!(EventType::NewParentView, is_null = true, height);
```

Comment:
Why remove these now? They were actually useful because they show up in `docker logs`, while looking at the debug logs currently is a bit of a PITA because of all the extra logs from 3rd party libraries like tower ABCI.

Reply:
I shifted its location, so that more info will be emitted